README.rst

```diff
@@ -36,16 +36,19 @@ Install ``openapi-common`` with:
 
 Usage
 ~~~~~
-The package exposes several classes, including a client ``ApiClient`` and a builder
-``ApiClientFactory``, these allow a client library to configure a connection to an API.
+The package exposes several classes, including a client :py:class:`~ansys.openapi.common.ApiClient` and a
```

Comment:
Should these be double backticks?

Reply:
This is the correct syntax for a link to a class, the class links work for me locally, on the other hand the method links do not. I will investigate.

Reply:
They did work, I had the wrong class.